examples/MagicWeather/app/build.gradle

```diff
@@ -48,8 +48,8 @@ android {
 }
 
 dependencies {
-    implementation 'com.revenuecat.purchases:purchases:$purchases_version'
-    implementation 'com.revenuecat.purchases:purchases-store-amazon:$purchases_version'
+    implementation "com.revenuecat.purchases:purchases:$purchases_version"
+    implementation "com.revenuecat.purchases:purchases-store-amazon:$purchases_version"
```

Comment:
Sorry to be picky but do you mind extracting these unrelated changes in MagicWeather to another branch? That way it's easier to revert and cherry pick changes :)

Reply:
yep no problem my bad!